swaap/swaap/Views/ProfileCardView.swift

```diff
@@ -13,6 +13,53 @@ import ChevronAnimatable
 @IBDesignable
 class ProfileCardView: IBPreviewView {
 
+	// MARK: - User Info Properties
+	var userProfile: UserProfile? {
+		didSet {
+			updateViews()
+		}
+	}
+
+	var profileImage: UIImage? {
+		get { profileImageView.image }
+		set { profileImageView.image = newValue }
+	}
+
+	var name: String? {
+		get { nameLabel.text }
+		set { nameLabel.text = newValue }
+	}
+
+	var tagline: String? {
+		get { taglineLabel.text }
+		set { taglineLabel.text = newValue }
+	}
+
+	var jobTitle: String? {
+		get { jobTitleLabel.text }
+		set { jobTitleLabel.text = newValue }
+	}
+
+	var location: String? {
+		get { locationLabel.text }
+		set { locationLabel.text = newValue }
+	}
+
+	var industry: String? {
+		get { industryLabel.text }
+		set { industryLabel.text = newValue }
+	}
+
+	var preferredContact: SocialLink? {
+		get { socialButton.socialInfo }
+		set {
+			guard let newValue = newValue else { return }
+			socialButton.socialInfo = newValue
```

Comment:
why is unwrapping necessary?

Reply:
The compiler was telling me it needs to be unwrapped